docs/src/index.md

```diff
@@ -299,47 +299,51 @@ function foo(x)
 end
 
 #### Find dfoo/dx via rrules
-
-# First the forward pass, accumulating rules
+#### First the forward pass, accumulating rules
 x = 3;
 a, a_pullback = rrule(sin, x);
 b, b_pullback = rrule(*, 2, a);
 c, c_pullback = rrule(asin, b)
 
-# Then the backward pass calculating gradients
+#### Then the backward pass calculating gradients
 c̄ = 1;  # ∂c/∂c
 _, b̄ = c_pullback(extern(c̄));     # ∂c/∂b
 _, _, ā = b_pullback(extern(b̄));  # ∂c/∂a
 _, x̄ = a_pullback(extern(ā));     # ∂c/∂x = ∂f/∂x
 extern(x̄)
-# -2.0638950738662625
-
+# output
+-2.0638950738662625
+```
+```jldoctest
```

Comment:
these need to be separate code blocks so that they're separate doctests, otherwise everything under `# output` is interpreted as output, including the next code example

Reply:
an altervative is to switch the the `julia>` repl style. But I think the current way is cleaner.